### ISSUE TITLE
learning to  boss joe

### ISSUE THREAD

**Issue by w-alone** [qiaosiyi/oj#1]
it's  a jiezou  of  paolu

**Comment by xuchen1992** [comment]
it's a jiezou of running away

**Comment by qiaosiyi** [comment]
> it's a jiezou of paolu

why do you think it is paolu, where to pao?

**Comment by qiaosiyi** [comment]
> it's a jiezou of running away

wow, what a surprise to see you here!

**Comment by w-alone** [comment]
> > it's a jiezou of paolu
> 
> why do you think it is paolu, where to pao?

yeah.it's right. U R one of the  xinfu  of boss hu. You don't need run away

**Comment by qiaosiyi** [comment]
> > > it's a jiezou of paolu
> > 
> > 
> > why do you think it is paolu, where to pao?
> 
> yeah.it's right. U R one of the xinfu of boss hu. You don't need run away

hello，good evening CEO 🐉
